README.md

```diff
@@ -1 +1,70 @@
-# ps_healthcheck
+# Health check
+
+## About
+
+Adds a block with several links.
```

Comment:
This module allows you to check if your instance of Prestashop have all the requirements to run correctly

... about configuring the security for the module ...

To have the instance status, call `http(s)://BASE_URL/ADMIN_DIR/modules/healthcheck`

... curl example with headers ...

The built in checks are
- connection to database
- read and write permissions for cache directory
- composer up to date

Some checks (like php version) are not made on purpose because without them the instance wont run at all

The module allows you to add your own checks